modules/nw_multinetwork-creating-first-attachments.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// * networking/managing_multinetworking.adoc
+
+[id='multinetwork-creating-first-attachments-{context}']
+= Creating additional network interfaces for the first time
+
+Additional interfaces for pods are defined in CNI configurations that are stored as Custom Resources.
```

Comment:
This first instance of Custom Resource should be defined as Custom Resources (CRs). All other instances on this page should be CR.